docs/i18n.rst

```diff
@@ -228,7 +228,13 @@ Two weeks before the release: string freeze
     consider pairing on this ceremony, both for knowledge-sharing and so that
     the intermediate pull requests can be reviewed and merged promptly.
 
-When features for a new SecureDrop release are frozen, the localization manager for the release will:
+When features for a new SecureDrop release are frozen, so are the source
+strings. The localization manager should apply any source-string suggestions
```

Comment:
Should `source-string` be hyphenated here?

Reply:
I believe so: "source-string" modifies "suggestions", rather than "source" modifying "string suggestions".